.github/workflows/codespell.yaml

```diff
@@ -14,5 +14,5 @@ jobs:
       - uses: codespell-project/actions-codespell@master
         with:
           only_warn: 1
-          ignore_words_list: "ser"
-          skip: "./.git,build/*,tools/*,*.js,*.thrift,test/*"
+          ignore_words_list: "ser,ue,crate"
```

Comment:
On the commit-msg you mentioned `ue` but not `crate` is it intentional? 
Other than that lgtm

Reply:
@benipeled hi, Beni, yeah, it's intentional. I found 'crate' in rust source code after adding ue, so added it as well.